perf/benchmark/runner/runner.py

```diff
@@ -281,24 +306,55 @@ def kubectl_cp(from_file, to_file, container):
     run_command_sync(cmd)
 
 
+def run_nighthawk(pod, remote_cmd, labels):
+    # Use a local docker instance of Nighthawk to control nighthawk_service running in the pod
+    # and run transforms on the output we get.
+    docker_cmd = "docker run --network=host {docker_image} {remote_cmd}".format(
```

Comment:
Is this running nh client to “nighthawk-service 127.0.0.1:{portmap}” ? 
Can you make a comment about the chain from docker onwards?

Reply:
Ugh, sorry about the confusion! I did `s/portmap/port forward/` everywhere.